tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
Hmmm... that's pretty smelly - I'll bet we can rework it to avoid state - serialization should be pure functions